R/textstat_simil.R

```diff
@@ -276,7 +276,10 @@ textstat_simil.dfm <- function(x, y = NULL, selection = NULL,
 
     x <- as.dfm(x)
     margin <- match.arg(margin)
-    method <- match.arg(method)
+    method <- match.arg(method, c(eval(formals(textstat_simil)[["method"]]), "hamman"))
+
+    # trap older "hamman" spelling
+    if (method == "hamman") method <- "hamann"
```

Comment:
How about doing this before above `match.arg(method)`. If so, `c(eval(formals(textstat_simil)[["method"]]), "hamman")` would be unnecessary.

Reply:
When I was fixing proxyC, I noticed that `method[method == "hamman"] <- "hamann"` works better.

Reply:
Obviously! Done.